moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -182,14 +182,14 @@ const DashboardView = () => {
       width: "10%",
     },
     {
-      title: "Name",
+      title: "Full name",
       field: "project.project_name",
```

Comment:
Small thing but I think this could be updated to `project_full_name` too in case we ever fall back to not using the custom `render`.

Reply:
you are absolutely right, I've updated them in the both places you pointed out.